Fix GetStates bug when there are multiple addresses (#346)
planetarium/libplanet: CHANGES.md

To be released.

### Bug fixes

>  -  Fixed a bug where the GetStates method does not return the latest state

Reviewer: ```suggestion
 -  Fixed a bug where the `BlockChain<T>.GetStates()` method had not returned
```

>     when there are multiple addresses.  [[#346]]

Reviewer: ```suggestion
    the latest state when there are multiple addresses.  [[#346]]
```
